hoomd/hpmc/integrate.py

```diff
@@ -311,28 +314,37 @@ def counters(self):
 
 
 class Sphere(_HPMCIntegrator):
-    R""" HPMC integration for spheres (2D/3D).
+    R""" Hard particle Monte Carlo integration method for spheres.
 
-    Args:
-        seed (int): Random number seed
-        d (float): Maximum move displacement, Scalar to set for all types, or a
-          dict containing {type:size} to set by type.
-        a (float): Maximum rotation move to set for all types, or a dict
-          containing {type:size} to set by type.
-        move_ratio (float, only used with **orientable=True**): Ratio of
-          translation moves to rotation moves. (added in version 2.3)
-        nselect (int): The number of trial moves to perform in each cell.
-        deterministic (bool): Make HPMC integration deterministic on the GPU
+    Sphere parameters:
 
-    Hard particle Monte Carlo integration method for spheres.
+    shape (particle type, dict): defines the shape of the object.
 
-    Sphere parameters:
+        Keys:
+            * *diameter* (float, **required**) - diameter of the sphere
+              (distance units)
+            * *ignore_statistics* (**default: False**) - set to True to ignore
+              tracked statistics
+            * *orientable* (**default: False**) - set to True for spheres with
+              orientation
+
+    d (particle type, float): the size of displacement trial moves
+
+    a (particle type, float): the size of rotation trial moves
+
+    fugacity (particle type, float): depletant fugacity (in units of density,
+    volume^-1)
 
-    * *diameter* (**required**) - diameter of the sphere (distance units)
-    * *orientable* (**default: False**) - set to True for spheres with
-      orientation (added in version 2.3)
-    * *ignore_statistics* (**default: False**) - set to True to disable ignore
-      for statistics tracking
+    interaction_matrix ((particle type, particle type), bool): whether to
+    include overlaps between type 1 and type 2
+
+    seed (int): random number seed
```

Comment:
We should have these before the `TypeParameters`. Otherwise it is confusing which are regular arguments. Also, it may be helpful to give the two different headings.

````
Attributes
-------------

Regular attributes seed, etc.

TypeParameters
----------------------

shape, etc.
````

Reply:
I realize this changes what I said earlier and you can leave the TypeParameter heading `Integrator Name parameters` if you like.

Reply:
Disregard comment. I disagree with my earlier opinion here.